build/modules/gke/cluster.tf

```diff
@@ -14,7 +14,7 @@
 
 provider "google-beta" {
   version = "~> 2.4"
-  zone      = "${lookup(var.cluster, "zone")}"
+  zone    = "${lookup(var.cluster, "zone")}"
```

Comment:
In the other file you changed places that had previously done a `lookup(...)` to a different syntax (`${var.cluster["zone"]}`) but this file still uses the `lookup(...)` syntax. Can you make this file consistent with `build/cluster.tf`?

Reply:
Yes, for sure